setup.py

```diff
@@ -2,11 +2,18 @@
 
 
 setup(
-    name="fms",
-    version="0.0.1",
-    author="Brian Vaughan, Joshua Rosenkranz, Antoni Viros i Martin, Davis Wertheimer, Supriyo Chakraborty, Raghu Kiran Ganti, ...",
-    author_email="bvaughan@ibm.com",
+    name="ibm-fms",
+    version="0.0.2",
+    author="Brian Vaughan, Joshua Rosenkranz, Antoni Viros i Martin, Davis Wertheimer, Supriyo Chakraborty, Raghu Kiran Ganti",
+    author_email="bvaughan@ibm.com, jmrosenk@us.ibm.com, aviros@ibm.com, Davis.Wertheimer@ibm.com, supriyo@us.ibm.com, rganti@us.ibm.com",
     description="IBM Foundation Model Stack",
+    url="https://github.com/foundation-model-stack/foundation-model-stack",
     packages=find_packages(),
     install_requires=["torch >= 2.1"],
+    extras_require={"hf": ["transformers >= 4.34.0"]},
```

Comment:
in our github workflows we now have: transformers==4.35.0 ... does this need to match that?